ui/components/ui/add-network/add-network.component.js

```diff
@@ -0,0 +1,116 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { I18nContext } from '../../../contexts/i18n';
+import ActionableMessage from '../actionable-message';
+import Box from '../box';
+import Typography from '../typography';
+import { COLORS, TYPOGRAPHY } from '../../../helpers/constants/design-system';
+
+const AddNetwork = ({
+  onBackClick,
+  onAddNetworkClick,
+  onAddNetworkManuallyClick,
+  featuredRPCS,
+}) => {
+  const t = useContext(I18nContext);
+
+  const onBack = () => {
+    onBackClick();
+  };
+
+  const onAddNetwork = () => {
+    onAddNetworkClick();
+  };
+
+  const onAddNetworkManually = () => {
+    onAddNetworkManuallyClick();
+  };
+
+  const nets = featuredRPCS
+    .sort((a, b) => (a.ticker > b.ticker ? 1 : -1))
+    .slice(0, 5);
+
+  return (
+    <Box className="add-network__wrapper">
+      <Box className="add-network__header">
+        <img
+          src="./images/caret-left-black.svg"
+          alt=""
+          onClick={onBack}
+          className="add-network__header__back-icon"
+        />
+        <h3>
+          <Typography variant={TYPOGRAPHY.H3} color={COLORS.BLACK}>
+            {t('addNetwork')}
+          </Typography>
+        </h3>
+      </Box>
+      <Box className="add-network__content">
+        <h6>
+          <Typography
+            variant={TYPOGRAPHY.H6}
+            color={COLORS.UI4}
+            margin={[4, 0, 0, 0]}
+          >
+            {t('addFromAListOfPopularNetworks')}
+          </Typography>
+        </h6>
+        <h7>
+          <Typography
+            variant={TYPOGRAPHY.H7}
+            color={COLORS.UI3}
+            margin={[4, 0, 3, 0]}
+          >
+            {t('customNetworks')}
+          </Typography>
+        </h7>
+        {nets.map((item) => (
+          <>
+            <Box className="add-network__content__featured-rpcs">
+              <img
+                className="add-network__content__featured-rpcs__token-image"
+                src={item.rpcPrefs.imageUrl}
+                alt=""
+              />
+              <h7>
+                <Typography variant={TYPOGRAPHY.H7} color={COLORS.BLACK}>
+                  {item.ticker}
+                </Typography>
+              </h7>
+              <img
+                className="add-network__content__featured-rpcs__add-icon"
+                src="./images/times.svg"
+                alt=""
+                onClick={onAddNetwork}
+              />
+            </Box>
+          </>
+        ))}
+      </Box>
+      <Box className="add-network__footer">
+        <h6 onClick={onAddNetworkManually}>
```

Comment:
NIT: This could rather be a button or a link.

Reply:
Fixed